x-pack/legacy/plugins/siem/public/components/formatted_date/index.test.tsx

```diff
@@ -162,4 +162,19 @@ describe('formatted_date', () => {
       });
     });
   });
+
+  describe('FormattedRelativePreferenceDate', () => {
+    describe('rendering', () => {
+      test('renders time over an hour correctly against snapshot', () => {
+        const isoDateString = '2019-02-25T22:27:05.000Z';
+        const wrapper = shallow(<FormattedRelativePreferenceDate value={isoDateString} />);
+        expect(wrapper.find('[data-test-subj="preference-time"]').exists()).toBe(true);
+      });
+      test('renders time under an hour correctly against snapshot', () => {
+        const timeTwelveMinutesAgo = new Date(new Date().getTime() - 12 * 60 * 1000).toISOString();
+        const wrapper = shallow(<FormattedRelativePreferenceDate value={timeTwelveMinutesAgo} />);
+        expect(wrapper.find('[data-test-subj="relative-time"]').exists()).toBe(true);
+      });
+    });
+  });
```

Comment:
Nice -- got both date isoDate and number dates! Here's a few extra bounds tests if you're interested 🙂 

``` ts
test('renders empty string value correctly', () => {
  const wrapper = mount(
    <TestProviders>
      <FormattedRelativePreferenceDate value={''} />
    </TestProviders>
  );
  expect(wrapper.text()).toBe(getEmptyString());
});

test('renders undefined value correctly', () => {
  const wrapper = mount(
    <TestProviders>
      <FormattedRelativePreferenceDate />
    </TestProviders>
  );
  expect(wrapper.text()).toBe(getEmptyValue());
});

test('renders null value correctly', () => {
  const wrapper = mount(
    <TestProviders>
      <FormattedRelativePreferenceDate value={null} />
    </TestProviders>
  );
  expect(wrapper.text()).toBe(getEmptyValue());
});
```